app/src/main/java/com/example/android/mashup/CreatorVideo/CreatorChooseVideoFragment.kt

```diff
@@ -71,8 +72,15 @@ class CreatorChooseVideoFragment : Fragment(), MashupClickListener {
             this@CreatorChooseVideoFragment
         );
 
+        val orientation = resources.configuration.orientation
+        val span = if (orientation == Configuration.ORIENTATION_LANDSCAPE){
```

Comment:
and here too

Reply:
To my knowledge the aren't ternary operators in Kotlin, the code you proposed is Java, however I did shorten the expression a bit 
`val span = if(orientation == Configuration.ORIENTATION_LANDSCAPE) 2 else 1`